aspnetcore/blazor/get-started.md

```diff
@@ -5,163 +5,114 @@ description: Get started with Blazor by building a Blazor app with the tooling o
-   dotnet new -i Microsoft.AspNetCore.Components.WebAssembly.Templates::3.2.0-rc1.20223.4
-   ```
-   
-   For information on the two Blazor hosting models, *Blazor WebAssembly* and *Blazor Server*, see <xref:blazor/hosting-models>.
-
-   > [!NOTE]
-   > The [.NET Core SDK version 3.1.201 or later](https://dotnet.microsoft.com/download/dotnet-core/3.1) is **required** to use the 3.2 Preview Blazor WebAssembly template. Confirm the installed .NET Core SDK version by running `dotnet --version` in a command shell.
-
 1. Install [Visual Studio Code](https://code.visualstudio.com/).
 
 1. Install the latest [C# for Visual Studio Code extension](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csharp) and the [JavaScript Debugger (Nightly)](https://marketplace.visualstudio.com/items?itemName=ms-vscode.js-debug-nightly) extension with `debug.javascript.usePreview` set to `true`.
 
-1. For a Blazor Server experience, execute the following command in a command shell:
+1. For a Blazor WebAssembly experience, execute the following command in a command shell:
 
    ```dotnetcli
-   dotnet new blazorserver -o WebApplication1
+   dotnet new blazorwasm -o WebApplication1
    ```
 
-   For a Blazor WebAssembly experience, execute the following command in a command shell:
+   For a Blazor Server experience, execute the following command in a command shell:
 
    ```dotnetcli
-   dotnet new blazorwasm -o WebApplication1
+   dotnet new blazorserver -o WebApplication1
    ```
 
+   For information on the two Blazor hosting models, *Blazor WebAssembly* and *Blazor Server*, see <xref:blazor/hosting-models>.
+
 1. Open the *WebApplication1* folder in Visual Studio Code.
 
 1. The IDE requests that you add assets to build and debug the project. Select **Yes**.
 
-1. With Blazor Server, run the app using the Visual Studio Code debugger.
+1. For a Blazor WebAssembly app, start the app using the **.NET Core Launch (Blazor Standalone)** launch configuration and then start the browser using the **.NET Core Debug Blazor Web Assembly in Chrome** launch configuration (requires Chrome). For more information, see <xref:blazor/debug#visual-studio-code>.
 
-   With Blazor WebAssembly, start the app using the **.NET Core Launch (Blazor Standalone)** launch configuration and then start the browser using the **.NET Core Debug Blazor Web Assembly in Chrome** launch configuration (requires Chrome). For more information, see <xref:blazor/debug#visual-studio-code>.
+   For a Blazor Server app, run the app using the Visual Studio Code debugger.
 
 1. In a browser, navigate to `https://localhost:5001`.
 
 # [Visual Studio for Mac](#tab/visual-studio-mac)
 
 > [!NOTE]
-> Blazor Server is supported in Visual Studio for Mac 8.4 or later, and Blazor WebAssembly support is included in Visual Studio for Mac 8.6 or later.
+> Blazor WebAssembly support is included in Visual Studio for Mac 8.6 or later, and Blazor Server is supported in Visual Studio for Mac 8.4 or later.
```

Comment:
Do we need this note? We don't have it above for VS. Instead can't we just tell folks to install the latest version of VS4Mac?

Reply:
Sayed asked for it. Ask him? ... or just change it to 'install latest'?

Reply:
I prefer to keep it simple and just tell folks to install latest. @sayedihashimi FYI.

Reply:
Sounds good